server/mainapp/models.py

```diff
@@ -123,3 +123,183 @@ def fetch_data(self) -> response.JsonResponse:
             return docs
 
         raise DataFetchingError("There Are No Posts In The Database At This Moment")
+
+
+class Likes:
```

Comment:
Also, If you check the user profile, it has a list for contact_us query IDs. Similarly, you should create a list for likes and comments so that we can see which posts a user has liked and commented. This will make user analysis and function handling easier.